packages/components/src/lib/d3/patchedScales.ts

```diff
@@ -73,7 +73,60 @@ function patchLinearishTickFormat<
 }
 
 function patchSymlogTickFormat(scale: ScaleSymLog): ScaleSymLog {
-  return patchLinearishTickFormat(scale);
+  // copy-pasted from https://github.com/d3/d3-scale/blob/83555bd759c7314420bd4240642beda5e258db9e/src/linear.js#L14
+  scale.tickFormat = (count, specifier) => {
+    const d = scale.domain();
+    return tickFormatWithCustom(d[0], d[d.length - 1], count ?? 10, specifier);
+  };
+  // UPSTREAM-ME: Patching symlog tick generator for better experience
+  scale.ticks = (count?: number) => {
+    if (count === 0) return [];
+
+    const [lower, upper] = scale.domain();
+    const c = scale.constant();
+
+    // We can't reliably use invert and transform from scale
+    // It converts relative to screen, and we would instead like it to be relative to 0
+    // const transform = scale
+    // const invert = scale.invert
+
+    function transform(x: number): number {
+      return Math.sign(x) * Math.log1p(Math.abs(x / c));
+    }
+
+    function invert(x: number): number {
+      return Math.sign(x) * Math.expm1(Math.abs(x)) * c;
+    }
+
+    /**
+     * @returns Closest number with a single significant digit being 1, 2 or 5
```

Comment:
I think this is still problematic.

First, the resulting tick can be out of bounds:
<img width="1497" alt="Captura de pantalla 2023-09-09 a la(s) 15 07 46" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/c2af5014-3d9a-4419-abf1-77b08c90a2dc">

Second, it can result in duplicate ticks, so in the end there will be only one unique tick.

It should round to the last significant digit based on the entire range (and also never go out of the range), not to the single topmost digit.

This is probably not that important, though, because it rarely makes sense to limit symlog in a small positive range.

Reply:
How should it had behaved on that range instead?

Reply:
Actually, I have an idea, will get down to implement it in a few hours.

Reply:
Welp, this looks better.

Also, I've returned boundary values, but this breaks here https://squiggle-website-git-cab-symlog-ticks-quantified-uncertainty.vercel.app/docs/Api/Plot